XSDElement/src/main/java/com/modelsolv/reprezen/gentemplates/xsdelement/xtend/PropertyRealizationHelper.xtend

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright © 2013, 2016 Modelsolv, Inc.
+ * All Rights Reserved.
+ *
+ * NOTICE: All information contained herein is, and remains the property
+ * of ModelSolv, Inc. See the file license.html in the root directory of
+ * this project for further information.
+ *******************************************************************************/
+
+package com.modelsolv.reprezen.gentemplates.xsdelement.xtend
+
+import com.google.common.collect.Iterables
+import com.modelsolv.reprezen.restapi.PropertyRealization
+import com.modelsolv.reprezen.restapi.ServiceDataResource
+import com.modelsolv.reprezen.restapi.TypedMessage
+import com.modelsolv.reprezen.restapi.datatypes.PrimitiveProperty
+import com.modelsolv.reprezen.restapi.datatypes.ReferenceProperty
+
+class PropertyRealizationHelper {
+	
+	extension FeatureHelper = new FeatureHelper
+
+	def private dispatch Iterable<PropertyRealization> getSequencePropRzs(ServiceDataResource dataResource) {
+		dataResource.includedProperties.getSequencePropRzs
+	}
+
```

Comment:
@andylowry wrote: 

> On the other hand, if getSequencePropRzs is likely to be a useful method in its own right for this set of utility classes, even if it's not used currently, then perhaps make the methods private and leave them as dispatch methods.

I think you meant "...perhaps make the methods _public_ and leave them as dispatch methods."

I stayed on the conservative / YAGNI side of the fence here, thinking it's better to introduce helper methods when they're actually needed, rather than carrying the extra weight of unused and (probably) untested methods. 

So I'm deleting the two unused overloads, keeping the one as a regular (single-dispatch) method, and moving it underneath the `hasSequencePropRzs` for clarity.